dipy/reconst/dti.py

```diff
@@ -1017,7 +1024,8 @@ def linearity(self):
 
         .. math::
 
-            Linearity = \frac{\lambda_1-\lambda_2}{\lambda_1+\lambda_2+\lambda_3}
+            Linearity =
+            \frac{\lambda_1-\lambda_2}{\lambda_1+\lambda_2+\lambda_3}
```

Comment:
Yes - the docs render just fine. I have found a few typos in the meanwhile, so adding the fixes for that.